run_on_azureml.py

```diff
@@ -1,24 +1,23 @@
-from argparse import ArgumentParser
 from pathlib import Path
 
+import fire
 from azureml.core import (
     Environment,
     Experiment,
-    Run,
     ScriptRunConfig,
     Workspace,
 )
 from azureml.core.compute import AmlCompute, ComputeTarget
 
 
 def find_or_create_compute_target(
-    workspace,
-    name,
-    instance_type="Standard_D8_v3",
-    min_nodes=0,
-    max_nodes=10,
-    idle_seconds_before_scaledown=900,
-    vm_priority="lowpriority",
+        workspace,
```

Comment:
hehe why the double tab? is this something your formatter did?

Reply:
seems so...